README.md

```diff
@@ -6,3 +6,11 @@
 
 
 More information at the [official website](https://spotbugs.github.io/). A lot of things can still be found at the [old FindBugs website](http://findbugs.sourceforge.net).
+
+# Build
+
+SpotBugs is built using Gradle. If you don't have Gradle installed, you can simply run the `gradlew` wrapper, which will automatically download Gradle if needed.
```

Comment:
actually, even with gradle installed, this is the recommended way. Otherwise, they would need a matching version of gradle (I think 3.1+ for the current config).